helm/ako/values.yaml

```diff
@@ -95,6 +95,21 @@ resources:
     cpu: 200m
     memory: 300Mi
 
+nodeSelector: {}
+
+tolerations: []
+
+affinity:
+  podAntiAffinity:
```

Comment:
@akshayhavile should we give the user the ability to change the affinity rules?

Reply:
hey @swathinsankaran in my use case I need to schedule AKO to specific worker nodes by setting conditions in affinity rule (e.g. egress firewall policy to the Avi controller)

Reply:
Thanks @alex1989hu. We will plan to include this in the upcoming release.